modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -94,38 +125,46 @@ To install the App via occ command, execute:
 
 === Configuring ClamAV within ownCloud
 
-IMPORTANT: If the app is enabled but either not configured or incorrectly configured it will *strictly reject all uploads* for the whole instance
+IMPORTANT: If the app is enabled but either not or incorrectly configured, it will *strictly reject all uploads* for the whole instance!
+
+ClamAV can be configured in following two ways:
 
-ClamAV can be configured in two ways:
+. xref:configure-clamav-using-the-av-configuration-panel[By using the Antivirus Configuration panel]
+. xref:configure-clamav-using-occ[By using the `occ config:app` command set.]
 
-. xref:configure-clamav-using-occ[By using the `occ config:app:set` command set.]
-. xref:configure-clamav-using-the-antivirus-configuration-panel[By using the Antivirus Configuration panel]
+==== Change Log Level Temporary
 
-==== Configure ClamAV Using the Antivirus Configuration Panel
+Once ClamAV is installed, select menu:Settings[General (Admin)] and, in the "*Log*" section, temporarily set btn:[Log level] to "_Everything (fatal issues, errors, warnings, info, debug)_".
```

Comment:
I guess that the original writer a) had only a "normal" install in his mind and no docker and b) wanted to highlight that a increased loglevel will help finding issues early because a non working av will block all uploads. Means from my pov at least a valid point. Ok if docker turns back the loglevel when restarting automatically., nothing to extra mention in my pov.

Reply:
Leaving as it is, can be improved in a later PR